index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@200;400;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/css/normalize.css">
+    <link rel="stylesheet" href="./css/style.css">
+
+</head>
+
+<body>
+    <nav class="nav flex">
+        <div>
+            <p>Already have an Intuit account?</p>
+            <p><a href="#"> I forgot my User ID or Password</a></p>
+        </div>
+        <button class="btn" type="submit"><span>&#128274;</span>Sign In</button>
+    </nav>
+    <header class="flex header">
+        <img src="./assets/images/Screenshot from 2020-07-14 10-24-13.png" alt="intuit logo">
+        <img src="./assets/images/mint_quick_turbo.png" alt="Mint Turbotax logo">
+    </header>
+    <main>
+        <form class="form">
+            <div class="form-padding flex">
+                <h1 class="center">Create an Intuit account</h1>
+                <p class="center intro zero-mgn">One account for everything Intuit, including Mint.</p>
+                <p class="center zero-mgn"><a href="#">Learn more</a></p>
+                <label for="email">Email Address</label>
+                <input type="email" autocomplete="email">
+                <label for="tel">Phone (recommended)</label>
+                <input type="tel" id="tel" autocomplete="tel">
+                <p class="small">Standard call, messaging or data rates may apply</p>
+                <label for="password">Password</label>
+                <input type="password" id="password" autocomplete="current-password">
+                <span class="pwd-key">&#128275;</span>
+                <button class="btn" type="submit"><span>&#128274;</span> Create Account</button>
+                <p class="consent zero-mgn">By clicking Create account, you agree to our
+                    Terms and have read and acknowledge our Privacy Statement.</p>
+            </div>
+            <p class="form-footer center">
+                Invisible reCAPTCHA by Google Privacy Policy and Terms of Use.
```

Comment:
- [ ] [OPTIONAL] The words `Privacy Policy` and `Terms of Use` should be web links as seen on the [original site](https://accounts.intuit.com/signup.html)